### DISCUSSION TITLE
Unable to pay Copilot

### DISCUSSION THREAD

**Discussion by berzezek** [community/community#33563] [Copilot Conversations] [answered]
I'm trying to pay copilot.
10 times it was deducted for 1 dollar, the payment did not go further, and access was not opened ...
You can help me?

**Comment by davecheney** [comment, marked as answer]
@berzezek please contact support who can assist you with payment issues. https://support.github.com/contact

**Comment by sridahrkatabathini** [comment]
showing error cords all cords
